cf_xarray/accessor.py

```diff
@@ -963,11 +965,16 @@ def describe(self):
         text += "\nCell Measures:\n"
         measures = self.cell_measures
         for key in sorted(self._get_all_cell_measures()):
-            text += f"\t{key}: {measures[key] if key in measures else []}\n"
+            if key in {**axes, **coords}:
+                text += f"{star}{key}: {measures[key] if key in measures else []}\n"
+            else:
+                text += f"\t{key}: {measures[key] if key in measures else []}\n"
 
         text += "\nStandard Names:\n"
         for key, value in sorted(self.standard_names.items()):
-            if key not in coords:
+            if key in {**axes, **coords, **measures}:
+                text += f"{star}{key}: {value}\n"
+            else:
                 text += f"\t{key}: {value}\n"
```

Comment:
Mmm maybe this is the most useful:
Show all keys and values in `axes`, `coordinates`, `cell_measures`, and `standard_names`.
Add a star to `cell_measures` and `standard_names` keys if they are also keys of `axes` and `coordinates` (and `cell_measures` for `standard_names`).